tensorflow/core/kernels/matmul_op_impl.h

```diff
@@ -218,6 +223,17 @@ struct SequentialMatMulKernel {
   }
 };
 
+inline Status FromExecutorStatus(const se::port::Status& s) {
```

Comment:
Isn't the returned `Status` and passed in `Status` the same type? What's the point of this function?